lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -104,30 +127,37 @@ const rule = (primary) => {
 
 						if (HAS_MIN_MAX_PREFIX.test(name)) {
 							// In a range context, min- and max- prefixed feature names are invalid
```

Comment:
[ask] Is this "In a range context" in the comment above incorrect (boolean context)?

Reply:
Yes, it is indeed incorrect.
I've changed this to :

```js
// In a boolean feature, min- and max- prefixed feature names are invalid
```